js/webusb-serial.js

```
@@ -951,6 +951,26 @@ class WebUSBSerial {
+        this._readLoopRunning = false;
+        
+        // Wait a bit for the read loop to finish
+        // The ReadableStream will close itself when _readLoopRunning becomes false
+        return new Promise((resolve) => {
+            setTimeout(() => {
+                // Create new streams
+                this._createStreams();
+                resolve();
+            }, 100);
+        });
+    }
```

Biome (2.1.2)

[error] Duplicate class member name "recreateStreams"



(lint/suspicious/noDuplicateClassMembers)